src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.ServiceConnection.json

```diff
@@ -268,6 +268,23 @@
         "AuthZ"
       ],
       "Enabled": true
+    },
+    {
+      "ControlID": "ADO_ServiceConnection_Dont_Grant_Broader_Group_Access_As_Approvers",
+      "Description": "Broader groups (contributors, project valid users, etc.) should not have excessive permissions on service connection.",
```

Comment:
Description: Broader groups (contributors, project valid users, etc.) should not be added as approvers on service connection.